frontend/src/queries/schema.ts

```diff
@@ -2521,3 +2522,33 @@ export enum AssistantGenerationStatusType {
 export interface AssistantGenerationStatusEvent {
     type: AssistantGenerationStatusType
 }
+
+export enum CustomChannelField {
+    UTMSource = 'utm_source',
+    UTMMedium = 'utm_medium',
+    UTMCampaign = 'utm_campaign',
+    ReferringDomain = 'referring_domain',
+}
+
+export enum CustomChannelOperator {
+    Exact = 'exact',
+    IsNot = 'is_not',
+    IsSet = 'is_set',
+    IsNotSet = 'is_not_set',
+    IContains = 'icontains',
+    NotIContains = 'not_icontains',
+    Regex = 'regex',
+    NotRegex = 'not_regex',
+}
```

Comment:
is this just the same as property operators? does it need to be its own declaration? or could be a `Pick<>` or `Omit<>`